css/style.css

```
@@ -490,20 +585,55 @@ img {
 }
 .templates-list {
   list-style: none;
+  padding: 0;
+  padding-top: 40px;
+  margin: 0;
+  display: grid;
+  grid-template-columns: 4fr 4fr;
+  grid-column-gap: 40px;
+  margin-bottom: 55px;
+  grid-row-gap: 70px;
```

Свойство `grid-gap` и его производные уже считаются устаревшими (вот так, только гриды появились, и...)
Это из-за того, что его сделали универсальным и будут использовать не только в гридах.
```suggestion
  column-gap: 40px;
  margin-bottom: 55px;
  row-gap: 70px;
```

---

Изменил

---

Ну это правильно, надо искать повторяющиеся ошибки: я же не буду в каждом прям месте писать )
Это и мне неудобно, и тебе непонятно зачем ревью, засоренное лишними комментами